manticore/core/state.py

```diff
@@ -360,7 +360,7 @@ def new_symbolic_value(self, nbits, label=None, taint=frozenset()):
         self._input_symbols.append(expr)
         return expr
 
-    def concretize(self, symbolic, policy, maxcount=7):
+    def concretize(self, symbolic, policy, maxcount=7, additional_symbolics=None):
```

Comment:
I'm not thrilled with introducing an additional argument to a user-facing function if it's only used in one other place in Manticore. I'd suggest either:
* Removing the `additional_symbolics` argument from `concretize` and instead calling `concretize` twice in ethereum/manticore.py and zipping the results (since it results in the same number of solver calls regardless)

_Or_

* Adding type hints and documentation items describing each argument, and making `additional_symbolics` keyword-only.

Reply:
Where I should add the type hint exactly for `additional_symbolics`?

Reply:
Right on this line :). So you'd get something that looks like `def concretize(self, symbolic, policy, maxcount=7, *, additional_symbolics: typing.Optional[typing.List[Expression]]=None):`